src/Components/Pagination/DefaultPagination.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Pagination } from '@mui/material'
+
+export type PaginationProps = {
+  totalPages: number
+  onChange: (event: React.ChangeEvent<unknown>, page: number) => void
+}
+
+const DefaultPagination = (props: PaginationProps) => {
+  return props.totalPages > 0 ? (
+    <Pagination count={props.totalPages} showFirstButton showLastButton boundaryCount={2} onChange={props.onChange} />
+  ) : (
+    <div />
+  )
```

Comment:
is it really necessary to return an empty div?


Maybe something like this works?

```suggestion
  return (props.totalPages > 0 && (
    <Pagination count={props.totalPages} showFirstButton showLastButton boundaryCount={2} onChange={props.onChange} />
  )
```